.changeset/giant-news-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Added support for updating TypeScript settings automatically when using `astro add`
```

Comment:
Would you mind expanding this description a bit more into like a paragraph so it can be copy/pasted into a release blog post? Thanks!

Reply:
Sure, let me do that